pdfly/cli.py

```diff
@@ -224,4 +238,5 @@ def x2pdf(
 metadata.__doc__ = pdfly.metadata.__doc__
 pagemeta.__doc__ = pdfly.pagemeta.__doc__
 compress.__doc__ = pdfly.compress.__doc__
+update_offsets.__doc__ = pdfly.update_offsets.__doc__
 x2pdf.__doc__ = pdfly.x2pdf.__doc__
```

Comment:
Those lines do not exist anymore on the `main` branch since mid-october.

Could you please rebase your branch @srogmann? 🙏

Reply:
I just performed the rebase on your branch.

@srogmann: `git pull --rebase` should give you an up-to-date version of the code 🙂

Reply:
I rebased.

    -@entry_point.command(name="x2pdf", help=x2pdf.update_offsets.__doc__)  # type: ignore[misc]
    +@entry_point.command(name="x2pdf", help=pdfly.x2pdf.__doc__)  # type: ignore[misc]

A copy-and-paste-type typo has been fixed.